source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -668,6 +716,8 @@ void ShaderGraph::finalize(ShaderGenerator& shadergen, const GenOptions& options
                         {
                             inputSocket = addInputSocket(interfaceName, input->type);
                             inputSocket->value = input->value;
+                            // Must map element path to a socket so it get's exposed properly.
+                            inputSocket->path = input->path;
```

Comment:
Nitpick: Maybe change the comment to: "Copy value and path from the internal input to the published socket"